server/gwt-editor/src/main/java/org/zanata/webtrans/client/ui/UnorderedListWidget.java

```diff
@@ -40,11 +40,13 @@ public void setDir(String dir) {
         ((UListElement) getElement().cast()).setDir(dir);
     }
 
+    @SuppressWarnings("deprecation")
     public void add(Widget widget) {
         // ComplexPanel requires the two-arg add() method
         super.add(widget, getElement());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Instead of suppressing these warnings, we should do what the deprecation messages say and use the other version.

You just need to upcast to `com.google.gwt.dom.client.Element` to choose the other overload.